hawc/apps/eco/forms.py

```diff
@@ -1,48 +1,147 @@
 from django import forms
 
-from ..common.admin import autocomplete
-from . import models
+from hawc.apps.common.forms import BaseFormHelper
+
+from ..common import selectable
+from . import lookups, models
 
 
 class DesignForm(forms.ModelForm):
+    CREATE_LEGEND = "Create new study design"
+    CREATE_HELP_TEXT = ""
+    UPDATE_HELP_TEXT = "Update an existing study design."
+
+    country = selectable.AutoCompleteSelectMultipleWidget(lookup_class=lookups.CountryLookup)
+    state = selectable.AutoCompleteSelectMultipleWidget(lookup_class=lookups.StateLookup)
+    ecoregion = selectable.AutoCompleteSelectMultipleWidget(lookup_class=lookups.EcoregionLookup)
+
     class Meta:
-        fields = "__all__"
+        exclude = ("study",)
         model = models.Design
-        widgets = {
-            "study": autocomplete(models.Design, "study"),
-            "countries": autocomplete(models.Design, "countries", multi=True),
-            "state": autocomplete(models.Design, "state", multi=True),
-            "ecoregion": autocomplete(models.Design, "ecoregion", multi=True),
-        }
+
+    def __init__(self, *args, **kwargs):
+        study = kwargs.pop("parent", None)
+        super().__init__(*args, **kwargs)
+        if study:
+            self.instance.study = study
+
+    @property
+    def helper(self):
+        for fld in ("habitat_as_reported", "climate_as_reported"):
+            self.fields[fld].widget.attrs["class"] = "html5text"
+            self.fields[fld].widget.attrs["rows"] = 3
+
+        if self.instance.id:
+            helper = BaseFormHelper(self)
+            helper.form_tag = False
+
+        else:
+            inputs = {
+                "legend_text": self.CREATE_LEGEND,
+                "help_text": self.CREATE_HELP_TEXT,
+                "cancel_url": self.instance.study.get_absolute_url(),
+                "submit_text": "Next",
+            }
+            helper = BaseFormHelper(self, **inputs)
+
+        helper.add_row("name", 3, "col-md-4")
+        helper.add_row("country", 3, "col-md-4")
+        helper.add_row("habitat", 4, "col-md-3")
+        return helper
 
 
 class CauseForm(forms.ModelForm):
+
+    term = selectable.AutoCompleteSelectField(
+        label="Cause Term",
+        lookup_class=lookups.NestedTermLookup,
+        required=False,
+    )
+
     class Meta:
-        fields = "__all__"
+        exclude = ("study",)
         model = models.Cause
-        widgets = {
-            "study": autocomplete(models.Cause, "study"),
-            "term": autocomplete(models.Cause, "term"),
-            "measure": autocomplete(models.Cause, "measure"),
-        }
+
+    def __init__(self, *args, **kwargs):
+        design = kwargs.pop("parent", None)
+        super().__init__(*args, **kwargs)
+        if design:
+            self.instance.study = design.study
+
+    @property
+    def helper(self):
+        for fld in ("comment", "as_reported"):
+            self.fields[fld].widget.attrs["class"] = "html5text"
```

Comment:
same here and elsewhere in the eco app, no html5text of wysiwyg editing (consistent w/ epiv2)